src/provider/application.js

```diff
@@ -334,6 +350,84 @@ class Application extends EventEmitter {
       this.trigger('xfc.unload');
     }
   }
+
+  /**
+   * When page load is completed, get the
+   * iframe's scrolling config, and set the tabIndex
+   * on the document.body of the embedded page if
+   * necessary.
+   */
+  handleLoadEvent() {
+    this.JSONRPC.request('isScrollingEnabled', [])
+      .then(this.setFocusWhenRequired);
+  }
+
+  /**
+   * Handle the resize event and check if tabIndex is needed to be set or removed.
+   * Depending on the content is scrollable or not, we need to update the tabIndex
+   * accordingly so focus isn't getting in the document when not needed.
+   */
+  handleResizeEvent() {
+    if (this.isIframeScrollable() === true && isContentScrollable() && this.hasInteractableElement === false) {
```

Comment:
Yeah, since we initialized to a boolean value, it is not needed to be strict. I updated the check with this commit:
https://github.com/cerner/xfc/pull/96/commits/8f4116da61e2b7259301f47330fb6438c028d925#diff-baaccc7106fb8a5ea9ea284ba7187c11cd3235ae46e0b917305efba4161c47c6R371